lib/iex/test/iex/helpers_test.exs

```diff
@@ -71,6 +71,7 @@ defmodule IEx.HelpersTest do
     cleanup_modules([Sample])
   end
 
+  @tag :does_not_apply_to_otp19
```

Comment:
Oh, wait. I see that this test actually compiles the files to beams in this test. Should be possible to make it work then.